sway/input/cursor.c

```diff
@@ -1222,6 +1231,9 @@ void handle_pointer_constraint(struct wl_listener *listener, void *data) {
 	sway_constraint->destroy.notify = handle_constraint_destroy;
 	wl_signal_add(&constraint->events.destroy, &sway_constraint->destroy);
 
+	sway_constraint->set_region.notify = handle_pointer_constraint_set_region;
+	wl_signal_add(&constraint->events.set_region, &sway_constraint->set_region);
```

Comment:
We need to `wl_list_remove` this listener in the constraint destructor handler.

Reply:
Missed that, thanks :+1: